e2e/__tests__/test_retries.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const runJest = require('../runJest');
+
+describe('Test Retries', () => {
+  const outputFileName = 'retries.result.json';
+  const outputFilePath = path.join(
+    process.cwd(),
+    'e2e/test-retries/',
+    outputFileName,
+  );
+
+  afterAll(() => {
+    fs.unlinkSync(outputFilePath);
+  });
+
+  it('retries failed tests if configured', () => {
+    let jsonResult;
+
+    const reporterConfig = {
+      reporters: [
+        ['<rootDir>/reporters/RetryReporter.js', {output: outputFilePath}],
+      ],
+    };
+
+    // Test retries only available via JEST_CIRCUS
+    // also testResults.invocations only available via JEST_CIRCUS
+    runJest(
+      'test-retries',
+      ['--config', JSON.stringify(reporterConfig), 'retry.test.js'],
+      {useJestCircus: true},
```

Comment:
you can do `skipOnJasmine` instead, then it will run just with circus

Reply:
https://github.com/facebook/jest/blob/d1ce3cd0135b5a8b64f84998fd5f36be4afb2bef/scripts/ConditionalTest.js#L17

Reply:
Ah perfect thanks!